site/content/docs/main/release-instructions.md

```diff
@@ -14,6 +14,32 @@ This page covers the steps to perform when releasing a new version of Velero.
 - Pre-releases: Any release leading up to a GA. Example: 1.4.0-beta.1, 1.5.0-rc.1
 - RC releases: Release Candidate, contains everything that is supposed to ship with the GA release. This is still a pre-release.
 
+# Velero Release Requirements
+
+Velero is on a "train leaves the station" model for releases.  We will generate a release candidate (RC)
+at the scheduled time.  Multiple release candidates may be generated, depending on if bugs are found during testing.
+When testing has passed a release build will be generated.
+
+## Release Candidate criteria
+The release candidate commit must meet the following criteria:
+
+* No major bugs outstanding 
+* Unit tests pass
+* E2E tests against latest K8S on AWS, vSphere pass
```

Comment:
Super nit but K8S is not an acronym, so it's weird to see it capitalized. Either s/K8S/K8s or s/K8S/k8s, thank you.

Reply:
Also, shouldn't Kind be on this list?

Reply:
Sure, Kind is a good idea.  They seem to have it as "kind" so that's what I'm putting in here.